backend/src/main/java/com/bakdata/conquery/resources/api/QueryResource.java

```diff
@@ -54,6 +58,24 @@ public FullExecutionStatus getStatus(@Auth Subject subject, @PathParam(QUERY) Ma
 		return processor.getQueryFullStatus(query, subject, RequestAwareUriBuilder.fromRequest(servletRequest), allProviders);
 	}
 
+	@GET
+	@Path("{" + QUERY + "}/statistics")
+	public Response getDescription(@Auth Subject subject, @PathParam(QUERY) ManagedExecution query) {
+
+		if (!(query instanceof ManagedQuery)) {
+			throw new BadRequestException("Statistics is only available for %s".formatted(ManagedQuery.class.getSimpleName()));
+		}
+
+		subject.authorize(query.getDataset(), Ability.READ);
+		subject.authorize(query, Ability.READ);
+
+		if(query.awaitDone(1, TimeUnit.SECONDS) != ExecutionState.DONE){
+			return Response.status(Response.Status.CONFLICT).build(); // Request was submitted too early.
```

Comment:
Ich glaube, dass du deinen Kommentar in die Response reinschreiben kannst. Das erleichtert das debuggen